File: leetcode/problemset/1.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        map<int, int> mp;
        vector<int> ans;
        for(int i = 0; i < nums.size(); i++) mp[target - nums[i]] = i;
        for(int i = 0; i < nums.size(); i++) {
            if(mp.count(nums[i]) && i != mp[nums[i]]) {
                ans.push_back(i);
                ans.push_back(mp[nums[i]]);
                return ans;
            }
        }
        return ans;
    }
};

int main() {
    return 0;
}

File: leetcode/biweekly-contest/97/1.py
class Solution(object):
    def separateDigits(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        return [int(x) for num in nums for x in str(num)]


s = Solution()
b = s.separateDigits([13,25,83,77])
pass
